OmniBLE/OmnipodCommon/BasalDeliveryTable.swift

```diff
@@ -170,58 +188,69 @@ public struct RateEntry {
     
     public var rate: Double {
         if totalPulses == 0 {
+            // Eros zero TB is the only case not using pulses
             return 0
         } else {
-            return round(TimeInterval(hours: 1) / delayBetweenPulses) / Pod.pulsesPerUnit
+            // Use delayBetweenPulses to compute rate, works for non-Eros near zero rates
+            return (.hours(1) / delayBetweenPulses) / Pod.pulsesPerUnit
         }
     }
     
     public var duration: TimeInterval {
         if totalPulses == 0 {
-            return delayBetweenPulses
+            // Eros zero TB case uses fixed 30 minute rate entries
+            return TimeInterval(minutes: 30)
         } else {
-            return round(delayBetweenPulses * Double(totalPulses))
+            // Use delayBetweenPulses to compute duration, works for non-Eros near zero rates
+            return delayBetweenPulses * totalPulses
         }
     }
     
     public var data: Data {
+        var ZZZZZZZZ = UInt32(delayBetweenPulses.hundredthsOfMilliseconds)
```

Comment:
Please use camel case naming convention for swift variables.

Reply:
The original name to describe this field was a bit ugly and actually less useful, but I'll change it back.